style.css

```diff
@@ -0,0 +1,136 @@
+body{
+    font-family: "Roboto" , sans-serif;
+    background: #ffffff;
+    width: 100%;
+    height: 100vh;
+    margin: 0;
+    color: #000000;
+    font-size: 15px;
+}
+.container {
+    width: 50%;
+    margin: 0 auto;
+    padding: 10px;
+}
+.items {
+    padding: 8px 48px;
+    margin: 8px;
+    border: 1px solid gray;
+    border-radius: 10px;
+}
+input[type="text"]{
+    outline: none;
+    width: 40%;
+    padding: 8px 25px;
+    border: 0px;
+    border-bottom: 1px solid #DDD;
+    margin: 15px;
+    font-size: 16px;
+}
+.items {
+    text-align:left;
+    border: 1px solid #DDD;
+    font-size: 15px;
+    font-family: "Roboto" , sans-serif;
+    font-weight: normal;
+    color: gray;
+}
+.items input{
+    margin: 0px 0px 0px 0px;
+    position: relative;
+    float:right;
```

Comment:
Floating is rarely used these days. Consider rewriting the item using flexbox. It is a lot easier.
Float can cause container collapsing as it will go out from the regular flow.
More information: https://www.linkedin.com/learning/css-essential-training-3/float-and-collapsed-container?autoplay=true